### ISSUE TITLE
Bug: Back button in the All Contributions tree map is not visible and doesn't work

### ISSUE THREAD

**Issue by adiati98** [open-sauced/app#3484]
### Describe the bug

There's an orange rectangle  in the All Contributions tree map in the Activity dashboard of Contributor Insights Page. I highlighted it and learned that it's a "Back" button.

There is no text available on the rectangle unless it's highlighted (on the desktop). Also, nothing happened when I click it.

#### Initial State

![Screenshot 2024-05-27 132844](https://github.com/open-sauced/app/assets/45172775/161fcc5a-8edf-4ea7-87fa-9ec4426fc265)

#### Highlighted

![Screenshot 2024-05-27 132704](https://github.com/open-sauced/app/assets/45172775/e1eb0de6-9819-47a1-b7c7-fb4df93cca8c)

### Steps to reproduce

1. Go to a Workspace that holds Contributor Insights pages.
2. Open a page in Contributor Insights.
3. Navigate to the "Activity" tab.
4. You will see an orange rectangle in the All Contributions tree map.
5. Highlight the rectangle. You will see a "Back" text.
6. Click it. It does nothing.

**Comment by nickytonline** [comment]
Thanks for reporting @adiati98. This isn't specific to the contributions treemap graph. It's an issue with the Button component for the `outline` variant when disabled.

See https://github.com/open-sauced/app/blob/06b73aff03fb95771ac1811ee15bd997e21651bd/components/shared/Button/button.tsx#L41

**Comment by nickytonline** [comment]
I'm going to revisit our buttons to get this sorted.
